### ISSUE TITLE
Allow more flexible disconnect options

### ISSUE THREAD

**Issue by la-oey** [hawkrobe/reference_games#23]
After last chatroom, if one player finishes the experiment and closes their tab before the other player has done so, the other player will be kicked off the game. This should be fixed by either making both players wait before closing out of tab (or within some other wait screen) or disconnect the servers before the final test trial.

**Comment by hawkrobe** [comment]
I agree there should be more control over when people get disconnected and what happens when they 'leave the experiment'! 

Just wanted to double-check why you closed it?

**Comment by la-oey** [comment]
I actually meant for it to be included as an issue for MH's generics version of the experiment, since we are currently struggling with this. I can re-open it though.
